kafka-ui-react-app/src/components/common/SmartTable/TableRow.tsx

```diff
@@ -61,21 +62,19 @@ export const TableRow = <T, TId extends IdType, OT = never>({
         if (!isColumnElement<T, TId>(child)) {
           return child;
         }
-        const { cell, field, width, className } = child.props;
-
+        const { cell, field, maxWidth, className } = child.props;
         const Cell = cell as React.FC<TableCellProps<T, TId, OT>> | undefined;
-
         return Cell ? (
-          <td className={className} style={{ width }}>
+          <Td className={className} style={{ maxWidth }}>
```

Comment:
put the browser td with additional class here instead of a component from a custom Class.